src/main/java/io/getunleash/util/UnleashScheduledExecutorImpl.java

```diff
@@ -48,7 +48,11 @@ public void setInterval(Runnable command, long initialDelaySec, long periodSec)
 
     @Override
     public Future<Void> scheduleOnce(Runnable runnable) {
-        return (Future<Void>) executorService.submit(runnable);
+        return executorService.submit(
```

Comment:
Think this was just wrong but didn't matter because the result wasn't really used. Pretty sure this is correct but still won't matter

Linter stops freaking though, which is nice